### ISSUE TITLE
GetSeason returning JikanValidationException for year bigger than current year

### ISSUE THREAD

**Issue by JaoHundred** [Ervie/jikan.net#24]
Hi Ervie, I've noticed GetSeason stopped working as expected, I'm not sure when this happened but the last version of jikan GetSeason returns JikanValidationException when selected year is bigger than current year.

**Steps to reproduce:**

- Pass any  year > current year to GetSeason

**Expected:**

- GetSeason returned data for future seasons if it exists

**Actual behavior:**

- GetSeason gives JikanValidationException at JikanDotNet.Helpers.Guard.IsValid[T] (System.Func`2[T,TResult] isValidFunc, T arg, System.String argumentName, System.String message) [0x00020] in <f159e0e535004b4f880f2705c990f462>:0 
  at JikanDotNet.Jikan.GetSeason (System.Int32 year, JikanDotNet.Seasons season) [0x0001d] in <f159e0e535004b4f880f2705c990f462>

**Comment by Ervie** [comment]
You're correct. I added some validation of input parameters to prevent sending http request that were going to throw exception anyway (like negative mal id etc.). I just checked various scenarios for seasons and it turned out that request with year from 1000 to 9999 are correct (just returning empty collection of animes). Everything up to 999 and above 10000 will return 400 Bad Request, so I will just extend the range.

Otherwise ValidationException should be thrown to keep "fail fast" approach. Will release next version soon and ntify you.
